src/UI/reference-ui/src/Layers/layersUIMachineOptions.js

```diff
@@ -2,15 +2,16 @@ import createLayersInterface from './createLayersInterface'
 import createLayerInterface from './createLayerInterface'
 import toggleLayerVisibility from './toggleLayerVisibility'
 import selectLayer from './selectLayer'
+import { idle, imageUpdating } from './imageUpdateIndicator'
 
 const layersUIMachineOptions = {
   layerUIActor: {
     actions: {
       createLayerInterface,
-
       selectLayer,
-
       toggleLayerVisibility,
+      imageUpdating,
+      idle,
```

Comment:
To enable the same behavior for point sets, geometry, can the `imageUpdating` and `idle` be updated to `startDataUpdate`, `finishDataUpdate`? Internally switch on the actor `type` like `toggleLayerVisibility`.